### PR TITLE
LYNX-155: Update documentation of queries/mutations to include introduces custom_attributes fields

### PR DESCRIPTION
**Pull request by loginesta**
## Purpose of this pull request

This pull request (PR) is to update documentation of queries/mutations to include introduced custom_attributes fields.

## Affected pages

src/pages/_includes/graphql/customer-address-input-24.md
src/pages/_includes/graphql/customer-address-output-24.md
src/pages/_includes/graphql/customer-output-24.md
src/pages/graphql/schema/customer/mutations/create-address.md
src/pages/graphql/schema/customer/mutations/create-v2.md
src/pages/graphql/schema/customer/mutations/update-address.md
src/pages/graphql/schema/customer/mutations/update-v2.md
src/pages/graphql/schema/customer/queries/customer.md

### CODE REVIEW

**Comment by loginesta** [comment]
@keharper Gonna push some changes by the end of the day. It's still WIP

**Comment by loginesta** [comment]
@keharper have already pushed the missing examples that I wanted to add. ready for review! thanks

File: src/pages/graphql/schema/customer/queries/customer.md [hunk 1]
```diff
@@ -76,6 +76,106 @@ The following call returns information about the logged-in customer. Provide the
 }
 ```
 
+### Retrieve custom attributes metadata from a customer
+
+The following call returns custom attributes for the logged-in customer. Provide the customer's token in the header section of the query.
```
Comment (keharper) [inline src/pages/graphql/schema/customer/queries/customer.md:81]: Add blank line below.

File: src/pages/graphql/schema/customer/queries/customer.md [hunk 2]
```diff
@@ -76,6 +76,106 @@ The following call returns information about the logged-in customer. Provide the
...
+        }
+      ]
+    }
+  }
+}
+```
+
+### Retrieve custom attributes metadata from a customer address
+
+The following call returns the customer address custom attributes for the logged-in customer. Provide the customer's token in the header section of the query.
```
Comment (keharper) [inline src/pages/graphql/schema/customer/queries/customer.md:143]: Add blank line below

File: src/pages/graphql/schema/customer/queries/customer.md [hunk 3]
```diff
@@ -925,6 +1026,36 @@ import CompareListOutput from '/src/pages/_includes/graphql/compare-list-output.
 
 <CompareListOutput />
 
+### AttributeValueInterface attributes
```
Comment (keharper) [inline src/pages/graphql/schema/customer/queries/customer.md:1055]: Add blank line below

File: src/pages/graphql/schema/customer/queries/customer.md [hunk 4]
```diff
@@ -938,6 +1069,7 @@ Attribute |  Data Type | Description
 `country_code` | CountryCodeEnum | The customer's country
 `country_id` | String | Deprecated. Use `country_code` instead. The customer's country
 `custom_attributes` | [CustomerAddressAttribute](#customeraddressattribute-attributes) | Deprecated. Not applicable for GraphQL
+`custom_attributesV2` | [AttributeValueInterface](#attributevalueinterface-attributes) | Customer address' custom attributes
```
Comment (keharper) [inline src/pages/graphql/schema/customer/queries/customer.md:1099]:
```suggestion
`custom_attributesV2` | [AttributeValueInterface](#attributevalueinterface-attributes) | Custom attributes assigned to the customer address
```

File: src/pages/graphql/schema/customer/queries/customer.md [hunk 5]
```diff
@@ -925,6 +1026,36 @@ import CompareListOutput from '/src/pages/_includes/graphql/compare-list-output.
...
+
+Currently, `AttributeValueInterface` has 2 different implementations: `AttributeValue` and `AttributeSelectedOptions`.
+
+Apart from the attributes described for `AttributeValueInterface`, the `AttributeValue` contains the following:
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`value` | String! | The attribute value
+
+On the other hand, the `AttributeSelectedOptions` contains the following attributes:
```
Comment (keharper) [inline src/pages/graphql/schema/customer/queries/customer.md:1072]:
```suggestion
The `AttributeSelectedOptions` object contains the following attributes:
```

File: src/pages/graphql/schema/customer/queries/customer.md [hunk 6]
```diff
@@ -925,6 +1026,36 @@ import CompareListOutput from '/src/pages/_includes/graphql/compare-list-output.
...
+The `AttributeValueInterface` contains the following attributes:
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`uid` | ID! | The unique ID of an attribute value
+`code` | String! | The attribute code
+
+Currently, `AttributeValueInterface` has 2 different implementations: `AttributeValue` and `AttributeSelectedOptions`.
+
+Apart from the attributes described for `AttributeValueInterface`, the `AttributeValue` contains the following:
```
Comment (keharper) [inline src/pages/graphql/schema/customer/queries/customer.md:1066]:
```suggestion
In addition to the attributes described for `AttributeValueInterface`, the `AttributeValue` contains the following:
```

File: src/pages/graphql/schema/customer/queries/customer.md [hunk 7]
```diff
@@ -925,6 +1026,36 @@ import CompareListOutput from '/src/pages/_includes/graphql/compare-list-output.
...
 
+### AttributeValueInterface attributes
+The `AttributeValueInterface` contains the following attributes:
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`uid` | ID! | The unique ID of an attribute value
+`code` | String! | The attribute code
+
+Currently, `AttributeValueInterface` has 2 different implementations: `AttributeValue` and `AttributeSelectedOptions`.
```
Comment (keharper) [inline src/pages/graphql/schema/customer/queries/customer.md:1064]:
```suggestion
Currently, `AttributeValueInterface` has two different implementations: `AttributeValue` and `AttributeSelectedOptions`.
```

**Review by keharper — CHANGES_REQUESTED** [review]
I recommend using the GH web interface to put all my grammatical edits in a batch. Then after merging that batch, pull the changes locally and make the remaining edits manually. It'll save time. Pardon if you know this already, but my experience is that most developers don't use GH this way.

**Review by keharper — CHANGES_REQUESTED** [review]
I recommend using the GH web interface to put all my grammatical edits in a batch. Then after merging that batch, pull the changes locally and make the remaining edits manually. It'll save time. Pardon if you know this already, but my experience is that most developers don't use GH this way.

File: src/pages/graphql/schema/customer/queries/customer.md (hunk 3 above)
Comment (loginesta) [reply src/pages/graphql/schema/customer/queries/customer.md:1057]: there is actually a blank line below. no need for an extra line.

**Review by keharper — APPROVED** [review]

**Comment by keharper** [comment]
Closing and reopening in hopes the automation detects that the Adobe CLA has been signed.

**Pull request merged**